docs/add_translation.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+import os
+
+
+def find_files(dir_path, suffix):
+    files = []
+
+    for root, _, filenames in os.walk(dir_path):
+        for filename in filenames:
+            if filename.endswith(suffix):
+                files.append(os.path.join(root, filename))
+    return files
+
+
+
+def exec(temp_directory):
+    files = find_files(temp_directory, '.rst')
+
+    for rst_file in files:
+        with open(rst_file, 'r+', encoding='utf-8') as f:
+            content = f.read()
+            f.seek(0, 0)
+            f.write(':link_to_translation:`zh_CN:[中文]`\n\n' + content)
```

Comment:
At the moment adding this hard-coded 'zh_CN' is OK, but during any new code development please think on code extendibility, too, if we want to re-use/extend it later; 
I mean in this case the language code could come from function parameter ( `exec(temp_directory, language_code)` ), so in the future it would be enough to call `exec()` again with different language, and the code adds the new language line.

Reply:
Good idea, other translations can be easily added in the future.
I will improve it in the new PR.